packages/blocks/src/api/registration.js

```diff
@@ -172,12 +173,15 @@ function getBlockSettingsFromMetadata( { textdomain, ...metadata } ) {
 		'variations',
 		'blockHooks',
 		'allowedBlocks',
+		blockMetadataDisplayKey,
 	];
 
 	const settings = Object.fromEntries(
-		Object.entries( metadata ).filter( ( [ key ] ) =>
-			allowedFields.includes( key )
-		)
+		Reflect.ownKeys( metadata )
```

Comment:
There is also the option to use a string as the key to avoid special handling, e.g., `__experimentalSomething`.

Reply:
Is it easier to make it a `supports` property?

Adding it to the root might also involve updating the json schema, whereas I think the `supports` allows anything.

Reply:
Wouldn't do any harm I think. I'll try it quickly.

Reply:
yeah, that's the ticket 👍🏻 Done